test/test_dmri.py

```diff
@@ -65,3 +140,55 @@ def test_load(datadir, tmp_path):
     assert np.allclose(dwi_h5.dataobj, dwi_from_nifti2.dataobj)
     assert np.allclose(dwi_h5.bzero, dwi_from_nifti2.bzero)
     assert np.allclose(dwi_h5.gradients, dwi_from_nifti2.gradients)
+
+
+def test_equality_operator(tmp_path):
+    # Create some random data
+    (
+        dwi_dataobj,
+        affine,
+        brainmask_dataobj,
+        b0_dataobj,
+        gradients,
+        fieldmap_dataobj,
+        b0_thres,
+    ) = _create_dwi_random_dataobj()
+
+    dwi, brainmask, b0, fieldmap = _create_dwi_random_data(
+        dwi_dataobj,
+        affine,
+        brainmask_dataobj,
+        b0_dataobj,
+        fieldmap_dataobj,
+    )
+
+    (
+        dwi_fname,
+        brainmask_fname,
+        b0_fname,
+        gradients_fname,
+        fieldmap_fname,
+    ) = _serialize_dwi_data(
+        dwi,
+        brainmask,
+        b0,
+        gradients,
+        fieldmap,
+        tmp_path,
+    )
+
+    lh_dwi_h5 = load(
+        dwi_fname,
+        gradients_file=gradients_fname,
+        b0_file=b0_fname,
+        brainmask_file=brainmask_fname,
+        fmap_file=fieldmap_fname,
+        b0_thres=b0_thres,
+    )
+
+    hdf5_filename = tmp_path / "test_dwi.h5"
+    lh_dwi_h5.to_filename(hdf5_filename)
+
+    rh_dwi_h5 = load(hdf5_filename)
```

Comment:
Intuitively, I saw "lh" and "rh" and thought hemispheres.

```suggestion
    dwi_obj = load(
        dwi_fname,
        gradients_file=gradients_fname,
        b0_file=b0_fname,
        brainmask_file=brainmask_fname,
        fmap_file=fieldmap_fname,
        b0_thres=b0_thres,
    )

    hdf5_filename = tmp_path / "test_dwi.h5"
    dwi_obj.to_filename(hdf5_filename)

    round_trip_dwi_obj = load(hdf5_filename)
```

Reply:
:+1:. Changed in a97f879.